src/emulator/gui/src/game_selector.cpp

```diff
@@ -179,22 +181,98 @@ void draw_game_selector(GuiState &gui, HostState &host) {
                     }
                     if (ImGui::IsMouseClicked(0))
                         selected[0] = true;
+                    if (gui.title_id != game.title_id)
```

Comment:
Understood. This check is here to prevent unnecessary copy. At the same time, this isn't you...

Reply:
yes, is for no change loop without needed ^^